kernel/idle.c

```diff
@@ -45,6 +45,8 @@ static void pm_save_idle(void)
 	if (pm_system_suspend(ticks) == false) {
 		k_cpu_idle();
 	}
+#else
+	k_cpu_idle();
 #endif
 }
```

Comment:
I think removing `pm_save_idle` would be even better, `static inline` feels wrong here.

Reply:
I re-oragnize it, though I don't see where it is wrong.